src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -77,14 +77,14 @@ class PressableWithSecondaryInteraction extends Component {
         // On Web, Text does not support LongPress events thus manage inline mode with styling instead of using Text.
         return (
             <Pressable
-                style={StyleUtils.combineStyles(this.props.inline ? styles.dInline : this.props.style)}
                 onPressIn={this.props.onPressIn}
                 onLongPress={this.props.onSecondaryInteraction ? this.executeSecondaryInteraction : undefined}
                 onPressOut={this.props.onPressOut}
                 onPress={this.props.onPress}
                 ref={(el) => (this.pressableRef = el)}
                 // eslint-disable-next-line react/jsx-props-no-spreading
                 {...defaultPressableProps}
+                style={StyleUtils.combineStyles(this.props.style, this.props.inline && styles.dInline)}
```

Comment:
added this at the end, since `defaultPressableProps` were overwriting the `style` prop when it was in the first line as `defaultPressableProps` have `this.props.style` as one of the keys. Here, we add display inline to default props, if `inline` is true.

Reply:
@Nikhil-Vats This led to regression for menu items on the Web and Desktop.

<details>
<summary>web</summary>
<img width="1079" alt="Screenshot 2023-05-25 at 21 34 44" src="https://github.com/Expensify/App/assets/25520267/9a3ebc84-37e0-48ac-8866-6f6701b2d484">



</details>

<details>
<summary>desktop</summary>

<img width="1206" alt="Screenshot 2023-05-25 at 21 36 38" src="https://github.com/Expensify/App/assets/25520267/941cc049-b2e6-4e95-b0e9-b0833a715dc5">

</details>

Reply:
Another regression on the copy url clipboard popover.

<details>
<summary>DEV</summary>

<img width="1080" alt="Screenshot 2023-05-25 at 21 45 31" src="https://github.com/Expensify/App/assets/25520267/e8679952-d48e-4b4c-b7d5-9fd7a933a847">

</details>

<details>
<summary>STAGING</summary>

<img width="1077" alt="Screenshot 2023-05-25 at 21 45 40" src="https://github.com/Expensify/App/assets/25520267/c0af16c1-cae4-4b65-abb1-7d51ca6c4cce">

</details>

Reply:
@mollfpr both fixed. I was not handling style props which were functions.